ssl-monitor/api/main.py

```
@@ -0,0 +1,350 @@
+    if domain and ip:
+        raise HTTPException(status_code=400, detail="Provide either domain or ip, not both")
+    
+    try:
+        result = check_single_target(domain=domain, ip=ip, port=port)
+        
+        # Save to database
+        check_id = save_check_result(db, result)
+        if check_id:
+            result["check_id"] = check_id
+        
+        return result
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)